Fix interrupt ctrl and data map

joelsa opened the pull request:
Now interrupts are working :tada:

qrasmont commented:
Thanks! I understand your frustration but could you reword your commit message to reflect what is actually fixed.

joelsa commented:
Yes, absolutely! I changed it to something more verbose.

I also adjusted the PR message above to something more cheerful.

Sorry, I am not really frustrated, this crate has already saved me quite some time versus writing it myself and I am just happy to see my interrupts working. :smiley:
